apps/ledger-live-mobile/src/screens/PairDevices/RenderError.tsx

```diff
@@ -41,6 +41,18 @@ const hitSlop = {
 
 function RenderError({ error, status, onBypassGenuine, onRetry }: Props) {
   const { colors } = useTheme();
+  const isPairingStatus = status === "pairing";
+  const isGenuineCheckStatus = status === "genuinecheck";
+  const isBrokenPairing = error instanceof PeerRemovedPairing;
+
+  const url = isBrokenPairing
+    ? urls.errors.PeerRemovedPairing
+    : (isPairingStatus && urls.errors.PairingFailed) || undefined;
```

Comment:
Why not use another ternary here? `isPairingStatus ? urls.errors.PairingFailed : undefined`

Reply:
because I probably just added what I had to the existing logic but sure, I guess